doc/develop/microscope.rst

```diff
@@ -91,6 +91,38 @@ Detectors:
 System:
  * lens: Contains parameters concerning the parabolic mirror and the lens system.
 
+Roles of components found in the CRYO/ENZEL system:
```

Comment:
in the cryoSECOM(ENZEL) system

Reply:
okay. i still struggle in distinguishing between the names